data/data/aws/main.tf

```diff
@@ -18,7 +18,7 @@ provider "aws" {
 module "bootstrap" {
   source = "./bootstrap"
 
-  ami                      = aws_ami_copy.main.id
+  ami                      = var.aws_ami
```

Comment:
Adjust the bootstrap module to always encrypt with the default KMS key?  Or share in `var.aws_master_root_volume_kms_key_id`?

Reply:
fixed in 9ab21a975fb5b5f36d32b05258be36068ddf9467
the bootstrap host is always encrypted like before, but uses default or user provided KMS key.